skip trailing comma for enums in TypeScript angular

bakito opened the pull request:
This change skips the trailing comma for enums in angular TypeScript models.

wing328 commented:
@bakito thanks for the PR. Does typescript-node need similar change? If yes, do you mind including that in this PR?

Also please update the Petstore sample as mentioned in the guidelines for contributing: https://github.com/swagger-api/swagger-codegen/wiki/Guidelines-for-Contribution

bakito commented:
I don't find any link between my commit and the build error that it caused.

Failed tests:   testCreateUsersWithArray(io.swagger.petstore.test.UserApiTest): expected:<1452772958870> but was:<1452772958808>

Has anybody an idea?

wing328 commented:
@bakito I'll look into the CI build error.

wing328 commented:
Restarted the CI job and the result is now ok.

bakito commented:
I've added the node template too.

I've also updated the petshop files.
The generated changes where mainly generation Timestamps and missing implementations of getPetByIdWithByteArrayWithCompletion addPetUsingByteArrayWithCompletion. I could not find any changes resulting of my commits.

Do I really have to commit these changes?

wing328 commented:
We usually update the sample as part of the PR. The new methods are recently added for testing the support of `binary` format.

I would recommend you to commit these changes if you've time.

bakito commented:
I committed the changes made in the angularjs and node modules, but without the binary stuff. I could not get the tests running with binary.

wing328 commented:
The binary stuff does not work likely due to the fact that there's no type mapping for `binary`. Please ignore that for the time being as we add support of `binary` language by language.

bakito commented:
My changes should be ok then, or is still something missing?

wing328 commented:
Looks good to me. Will take another look tomorrow and merge accordingly.

wing328 commented:
PR merged. Thanks for the contribution.
